powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/util/ResultStatusUtil.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * PowerAuth test and related software components
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.security.powerauth.app.testserver.util;
+
+import com.wultra.security.powerauth.app.testserver.database.TestStatusRepository;
+import com.wultra.security.powerauth.app.testserver.database.entity.TestStatusEntity;
+import org.json.simple.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+/**
+ * Utility service for persistence of result status.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Service
+public class ResultStatusUtil {
```

Comment:
The naming should be ideally `*Service`.

Furthermore I would implement the `ResultStatusService` interface here. As this is just another alternative persistence way for result status object.